ui/app/routes/vault/cluster/secrets/backend/secret-edit.js

```diff
@@ -301,6 +302,7 @@ export default Route.extend(UnloadModelRoute, {
     },
 
     willTransition(transition) {
+      /* eslint-disable-next-line ember/no-controller-access-in-routes */
```

Comment:
gosh we use controllerFor in loads of places.  Do you know when this will be officially depreciated?

Reply:
That's a good question, I don't know if it's a disallow thing or a best practice thing 🤔 Cursory search did not bring up any answers but maybe we should ask in the front end sync